README.md

```diff
@@ -1 +1,16 @@
-# bpanel wallet
+# bPanel bwallet
+
+Work in progress.
+
+### Usage
+
+Use as a plugin with `bPanel` to access your blockchain nodes.
+In `github.com/bpanel/bpanel`, update the config file
+so that the plugins array contains `@bpanel/simple-wallet`.
```

Comment:
I think this text needs to be updated? It would be to interact with your wallets not your nodes. also the github url isn't right and you would want bwallet, not simple-wallet. 

I think it's worth pointing users towards bpanel-cli for plugin installation now too.

Reply:
Will update with `bpanel-cli` info and update the text as well

Reply:
👍